codecov.yml

```diff
@@ -0,0 +1,20 @@
+# Documentation:  https://docs.codecov.io/docs/codecov-yaml
+ignore:
+  - "api/v1alpha1/zz_generated.deepcopy.go" # generated file
+  - "integration_tests"                     # integration tests
+  - "test_util.go"                          # test util
+  - "testtokenstorage.go"                   # test util
+
+coverage:
+  status:
+    # Allows coverage to drop by a 3% when compared against the base commit.
+    project:
+      default:
+        target: auto
```

Comment:
I'm not sure about this, it basically allows us to decrease coverage by 3% by every commit. We're on ~65% now, which to me is quite low, what about start with 65%?